Sources/EmbraceCrash/EmbraceCrashReporter.swift

```diff
@@ -49,10 +55,16 @@ public final class EmbraceCrashReporter: NSObject, CrashReporter {
             return
         }
 
-        ksCrash.userInfo = [
-            UserInfoKey.sdkVersion: sdkVersion ?? NSNull(),
-            UserInfoKey.sessionId: currentSessionId ?? NSNull()
-        ]
+        if ksCrash.userInfo == nil {
```

Comment:
May want to build up a temporary dictionary here and then set that on `ksCrash.userInfo` in one go. Hoping to make this interaction more atomic in case a crash occurs when this is occurring.

```swift
var info = extraInfo
info[UserInfoKey.sdkVersion] = sdkVersion ?? NSNull()
info[UserInfoKey.sessionId] = currentSessionId ?? NSNull()

ksCrash.userInfo = info
```